cmd/registry/googleauth/client.go

```diff
@@ -42,8 +42,7 @@ func GetAuthenticatedClient(ctx context.Context) (*http.Client, error) {
 	}
 	// Create configuration for specified scopes.
 	// If you modify these scopes, delete your previously saved token.
-	config, err := google.ConfigFromJSON(b,
-		"https://www.googleapis.com/auth/spreadsheets")
+	config, err := google.ConfigFromJSON(b, "https://www.googleapis.com/auth/spreadsheets")
```

Comment:
Did an automatic formatter make this change?

Reply:
This shouldn't have been included in the PR.